custom-assembly-report-generator/get_assembly_report_url.py

```diff
@@ -0,0 +1,41 @@
+from ftplib import FTP
+import argparse
+
+
+def get_assembly_report_url(assembly_accession):
+    if len(assembly_accession) != 15:

```

Comment:
I realised this week that versions can be 2 digits. I was overly simplistic when I coded this. for instance, human is currently on "GCA_000001405.27". Change the message as well